existing-communities/README.md

```diff
@@ -1 +1,47 @@
-# Existing Communities
+# Communities
+A community is, according to the Oxford Dictionary,  *a group of people living in the same place or having a particular characteristic in common*{{"ComDef" | cite}}. In traditional research, the study focuses on localy bound communities like neighbourhoods and villages, because they have clear boundaries of who is involved in the community{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.  
+But thanks to modern media, like games, social media and messenger-applications, communities are not bound anymore by location and the characteristic becomes the central point of the community. A concequence of this is that boundaries of the community are less clear. Therefor, one of the challenges of doing research on these virtual communities is determining who is and who is not part of a community{{"sundaram2012understanding" | cite}}.  
```

Comment:
`Therefor, one of the challenges`

I think you mean therefore? http://grammarist.com/usage/therefore-vs-therefor/